1-exercises/E-for-loop/exercise2.js

```diff
@@ -26,6 +26,10 @@ const AGES = [
     49
 ];
 
+for(let i=0;i<WRITERS.length;i++){
+    console.log(`${WRITERS[i]} is ${AGES[i]} years old`);
+}
+
```

Comment:
Yes, good use of the loop index for both arrays.

Reply:
Thank you